www/postfixadmin/Portfile

```diff
@@ -36,8 +35,41 @@ destroot {
     file copy {*}[glob ${workpath}/${distname}/*] ${docpath}/html
 }
 
+default_variants    +mysql
+
+variant mysql description "use mysql8 for the db backend" {
+    depends_run-append \
+                    port:mysql8 \
+                    port:php73-mysql
+} 
+
+variant mariadb description "use mariadb-10.2 for the db backend" {
+    depends_run-append \
+                    port:mariadb-10.2 \
+                    port:php73-mysql
+} 
+
+variant percoan description "use percona for the db backend" {
+    depends_run-append \
+                    port:percona \
+                    port:php73-mysql
+} 
+
+variant postgresql description "use postgresql11 for the db backend" {
```

Comment:
Given all of the versions of postgresql, maybe this should be named based on the version - `postgresql11`. I believe other ports with postgresql variants do this. Look at `qgis3` for example.